wca-guidelines.md

```diff
@@ -205,10 +205,16 @@ To be more informative, each Guideline is classified using one of the following
 
 ## <article-E><fewest-moves><fewestmovessolving> Article E: Fewest Moves Solving
 
+- E2a1+) [RECOMMENDATION] If a judge finds a submitted solution that does not appear to have information to identify the attempt, they should check both sides of the paper.
```

Comment:
I don't think E2a1 is necessary - that kind of is just what you'd do regardless of this reg in place or not?

Reply:
I think we need to be explicit about what is allowed to avoid inconsistent rulings from Delegates. The incidents from this year suggest that not every Delegate is on the same page about this right now. :-(